up-board.coffee

```diff
@@ -0,0 +1,57 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+UPBOARD_FLASH = '''
+	Power up the <%= TYPE_NAME %>. Make sure you have a keyboard connected.
+	Press the F7 key while BIOS is loading in order to enter the boot menu. Next, select the "UEFI:" option.
+'''
+
+UPBOARD_POWERRESET = 'Plug and unplug power on you <%= TYPE_NAME %>.'
```

Comment:
And should it not be the other way around? Like: unlug and plug? Instead of plug and unplug?